Names

IohannesIohannium opened the pull request:
- Reviewed and localised almost all names I:R adds (I have yet to check the Gargantuan addition to the Byzantine cultures)
-- If somebody wishes to review the Russian, Chinese and Korean trans_litterations_, they're welcome to correct any error they find
- Fixed the Roman naming pool so it works as it historically did with the tria nomina

IhateTrains commented on ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_central_germanic.txt:
```diff
@@ -1384,6 +1384,20 @@
 			25 = caucasian_dark_hair
 		}
+		
+		dynasty_of_location_prefix = "dynnp_von"
```
If we're adding this, isn't "fan" more fitting? https://en.wiktionary.org/wiki/Reconstruction:Proto-Germanic/fan%C4%93 "Von" sounds too High German for Vandals.

IohannesIohannium replied:
I was planning to initially localize all names and dynasties, and then look back to patronymics and "de Townsville". I used _von_ as a starting fix, but I'll use _fanē_ for Vandal, _af_ (https://en.wiktionary.org/wiki/%F0%90%8C%B0%F0%90%8D%86) for Gothic.

IhateTrains commented on ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_central_germanic.txt:
```diff
@@ -1469,6 +1485,20 @@
 			25 = caucasian_dark_hair
 		}
 
+		dynasty_of_location_prefix = "dynnp_von"
```
Ditto

IhateTrains commented on ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_latin.txt:
```diff
@@ -296,6 +311,11 @@
 		}
+		
+		mercenary_names = {
+			{ name = "mercenary_company_legion" coat_of_arms = "e_roman_empire" }
```
Legions shouldn't really be mercenaries. When I get to converting armies, I intend to convert Imperator Legions (the 2.0 professional armies, not just Roman legions) to men-at-arms. Vanilla seems to agree with me.
![image](https://user-images.githubusercontent.com/29546927/102080719-3801b680-3e0f-11eb-9b81-0779327b6caa.png)

IohannesIohannium replied:
I meant it to be a mercenary army specific to the Latin culture. Perhaps _Legio XXXII Mercenaria_ would be a better localisation, which I'll add along with the _fanē_ and the _af_

IhateTrains replied:
I think I have an idea for a better mercenaries localization: _Numerus_
> Numeri were allied native (or "barbarian") units from outside the Empire who fought alongside the regular forces on a mercenary basis. These were led by their own aristocrats and equipped in traditional fashion. Numbers fluctuated according to circumstances and are largely unknown.

https://en.wikipedia.org/wiki/Numerus_(Roman_military_unit)

IhateTrains commented on ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_latin.txt:
```diff
@@ -235,53 +240,63 @@
-		dynasty_of_location_prefix = "dynnp_of"
-		patronym_prefix_male = "dynnpat_pre_filius"
+		
+		dynasty_of_location_prefix = "dynnp_a"
```
What is the prefix based on?

IohannesIohannium replied:
If it were possible to use suffixes, I'd have a place called _Gießen_ and have _Gießenensis_. But since this can't be, _a Gießen_ ("from Gießen") is a valid enough alternative. _A(b)_ as a preposition is sometimes used in Latin to translate the _de X_ and the _von X_.

IhateTrains commented on ImperatorToCK3/Data_Files/blankMod/output/localization/ir_names_l_english.yml:
```diff
@@ -0,0 +1,1940 @@
+ dynn_Galid:0 "Galid"
+ dynn_Jugurthid:0 "Jugurthid"
+ 
+ mercenary_company_legion:0 "Roman Legion"
```
This should probably be in a separate file.

IohannesIohannium replied:
Agreed, I'll solve it in the next pull where I localize also the names of Byzantine converter cultures.

IhateTrains commented on ImperatorToCK3/Data_Files/blankMod/output/common/culture/cultures/00_central_germanic.txt:
```diff
@@ -1384,6 +1384,20 @@
+		pat_grf_name_chance = 50
+		mat_grf_name_chance = 5
+		father_name_chance = 10
+		patronym_suffix_male = "dynnpat_suf_son"
```
Any basis for this?

IohannesIohannium replied:
None at all, see comment above about the Vandal _von/fanē_

IohannesIohannium commented:
- Finished the job also with new Byzantine cultures
- Corrected patronymics and "de X" for various cultures
- Added nickname mappings for some Roman cognomina
- Renamed the Roman mercenaries "Numeri"

IhateTrains reviewed (commented):
There are some duplicated localization files (for example `ImperatorToCK3/Data_Files/blankMod/output/localisation/ir_names_l_english.yml` and `ImperatorToCK3/Data_Files/blankMod/output/localization/ir_names_l_english.yml`). `localization` is the correct folder name, but the `localisation` files were added later. Can I just (after merging the PR) remove the `localization` ones and copy the `localisation` ones to the `localization` folder?

IohannesIohannium commented:
Yeah, do that. I blame English spelling.

IhateTrains commented:
> Yeah, do that. I blame English spelling.

Done #107
Also renamed mercenary_company_legion to mercenary_company_numeri to be consistent with the loc.
